blockchain/network.py

```diff
@@ -421,10 +399,10 @@ def phase_2_broadcast(self, block_info, phase_type):
 
         phase_2_msg = message_types.Phase_2_msg()
         phase_2_msg.record = get_verification_record(verification_record)
-        phase_2_msg.valid_txs = map(convert_to_thrift_transaction, verification_info['valid_txs'])
-        phase_2_msg.invalid_txs = map(convert_to_thrift_transaction, verification_info['invalid_txs'])
-        phase_2_msg.business = verification_info['business']
-        phase_2_msg.deploy_location = verification_info['deploy_location']
+        phase_2_msg.valid_txs = map(convert_to_thrift_transaction, verification_info['VALID_TXS'])
+        phase_2_msg.invalid_txs = map(convert_to_thrift_transaction, verification_info['INVALID_TXS'])
+        phase_2_msg.business = verification_info['BUSINESS']
```

Comment:
Understood, and thank you for the direction on this.  
- [ ] This will be rectified. :persevere: